src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,31 @@
+import { Person } from '../../types';
+import { PersonItem } from '../PersonItem';
+
+type Props = {
+  people: Person[]
+};
+
+export const PeopleList = ({ people }: Props) => {
+  return (
+    <tbody>
+      {people.map(person => {
+        const { slug, motherName, fatherName } = person;
+
+        const personMother = people
+          .find(({ name }) => name === motherName);
+
+        const personFather = people
+          .find(({ name }) => name === fatherName);
```

Comment:
Better to prepare your people array after API response and add `mother` and `father` to every object